exporter/elasticsearchexporter/config.go

```diff
@@ -103,6 +103,9 @@ type BatcherConfig struct {
 type TelemetrySettings struct {
 	LogRequestBody  bool `mapstructure:"log_request_body"`
 	LogResponseBody bool `mapstructure:"log_response_body"`
+
+	LogFailedDocsSource          bool          `mapstructure:"log_failed_docs_source"`
+	LogFailedDocsSourceRateLimit time.Duration `mapstructure:"log_failed_docs_source_rate_limit"`
```

Comment:
[to reviewer] no strong on opinion on nested vs flat configs

Reply:
Flat looks good to me for this use case but no strong opinions either.